aws.sh

```diff
@@ -8,10 +8,18 @@ set -o pipefail
 trap - INT TERM
 
 docker run --rm \
-	-t $(tty &>/dev/null && echo "-i") \
-	-e "AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}" \
-	-e "AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}" \
-	-e "AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}" \
+	-t "$(tty >/dev/null 2>&1 && printf -- '-i')" \
+	-v "${AWS_SHARED_CREDENTIALS_FILE:-$HOME/.aws}:${AWS_SHARED_CREDENTIALS_FILE:-/root/.aws}" \
+	${AWS_ACCESS_KEY_ID:+-e "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID"} \
```

Comment:
Those are a bit verbose but here's the idea, if `AWS_ACCESS_KEY_ID` is defined, then we output `-e "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID"` otherwise we skip it.

Some environment variables if set (but empty) will cause the CLI to misbehave, so we need to make sure that the env variable is set before propagating it